ui/src/containers/NodeVolumes.js

```diff
@@ -87,6 +91,12 @@ const LoaderContainer = styled(Loader)`
   padding-right: ${padding.smaller};
 `;
 
+const TrashButtonContainer = styled(Button)`
+  ${props => {
+    if (props.disabled) return { opacity: 0.2 };
```

Comment:
Nothing to change here but we might want to discuss about the opacity for disabled stuff with Valamir.